duckduckgo/iOS
# Add icons for Google and Facebook child entities.

Task/Issue URL: https://app.asana.com/0/0/1202836875374614/f
Tech Design URL:
CC:

**Description**:
Adds icons for Google and Facebook child entities (added in the next blocklist update).



**Steps to test this PR**:
1. Switch to the 'beta' blocklist (change 'v3' to 'beta' [here](https://github.com/duckduckgo/iOS/blob/develop/Core/AppUrls.swift#L39)).
2. Visit test sites. Verify that for the new entities the correct icon is shown:
  - "Google Ads (Google)": youtube.com, android.com
  - "Google Analytics (Google)": blogger.com
  - "Youtube (Google)": https://privacy-test-pages.glitch.me/privacy-protections/youtube-click-to-load/ (in not blocked section).
  - "Instagram (Facebook)": https://www.cinemablend.com/movies/see-the-four-hobbits-from-lotr-reuniting-at-a-restaurant-and-totally-making-the-day-of-a-nearby-fan 



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
